toqito/state_opt/state_distinguishability.py

```diff
@@ -1,12 +1,15 @@
 """State distinguishability."""
-import cvxpy
 import numpy as np
+import picos
 
-from .state_helper import __is_probs_valid, __is_states_valid
+from toqito.state_ops import pure_to_mixed
 
 
 def state_distinguishability(
-    states: list[np.ndarray], probs: list[float] = None, dist_method: str = "min-error"
+    vectors: list[np.ndarray],
+    probs: list[float] = None,
+    solver: str = "cvxopt",
+    primal_dual: str = "dual",
```

Comment:
Since you have removed `dist_method`, you should remove the options for it from the docstring. 

Stuff under `math::` will also change due to this.

Reply:
Yep, good point. Changed!